public/src/world/item.js

```
@@ -1,10 +1,14 @@
 import * as THREE from 'three';
-import { Item as ItemConst } from '../utils/constants.js';
 
 export class Item {
-    constructor(type, position) {
+    constructor(type, position, game) {
         this.type = type;
-        const geometry = new THREE.SphereGeometry(ItemConst.SPHERE_RADIUS, ItemConst.GEOMETRY_SEGMENTS, ItemConst.GEOMETRY_SEGMENTS);
+        const itemData = game.data.items?.generic; // Add defensive check
+        if (!itemData) {
+            console.warn('Item data for generic not found. Using default values.');
+            itemData = { PICKUP_RANGE: 0.5, SPHERE_RADIUS: 0.2, GEOMETRY_SEGMENTS: 8 };
```

Biome (1.9.4)

[error] Can't assign itemData because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)